sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -324,7 +325,10 @@
    <version ClassVersion="10" checksum="2672302824"/>
   </class>
   <class name="std::vector<caf::SROpFlash>" />
-  
+  <class name="caf::SRCRTPMTMatch"/>
```

Comment:
Please enable the ROOT I/O schema evolution for this class:
```suggestion
  <class name="caf::SRCRTPMTMatch" ClassVersion="10" />
```
After that change, make sure to rerun the build, so that ROOT will add the actual checksum to this file (`SROpFlash` above should also have it... apparently it slipped).